src/Console/Application.php

```diff
@@ -88,6 +141,11 @@ public function getContainer(): Container
         return $this->container;
     }
 
+    public function getIO(): SymfonyStyle
```

Comment:
I'm not convinced this is very useful here, a `$io` variable looks like to be enough for the case above

Reply:
SymfonyStyle is using in the InfectionCommand so it's preventing of creating 2 instances of the same object